include/RAJA/util/Operators.hpp

```diff
@@ -386,7 +386,7 @@ struct logical_and : public detail::comparison_function<Arg1, Arg2>,
   {
     return lhs && rhs;
   }
-  static constexpr const bool identity = true;
+  static constexpr bool identity() {return true;}
```

Comment:
I asked because some operators have an identity and some do not and it's clear what identity means for some (+, *, etc.), but it may not be for others (I can see why identity for && is 'true', but 'false' for || seems arbitrary)?  I'm fine with this if it makes sense to others.